src/core/core.cpp

```diff
@@ -72,3 +72,12 @@ std::vector<User> getUsers(const json& jsonData) {
 
   return users;
 }
+
+bool getLogin(const std::string &username, const std::string &password){
```

Comment:
Not sure if `getLogin` is a good name, it suggest like we are acquiring a login from somewhere but it does not return a login but a `bool` which, on the other hand, suggest that the function check something. I'd change the name.